lib/compute/network.js

```diff
@@ -215,6 +221,137 @@ Network.prototype.createFirewall = function(name, config, callback) {
 };
 
 /**
+ * Create a subnetwork in this network.
+ *
+ * @resource [Subnetwork Resource]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource}
+ * @resource [Subnetwork: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/subnetworks/insert}
+ *
+ * @param {string} name - Name of the subnetwork.
+ * @param {object} config - See a
+ *     [Subnetwork resource](https://cloud.google.com/compute/docs/reference/v1/subnetworks#resource).
+ * @param {string=} config.description - An optional description of this
+ *     resource. Provide this property when you create the resource.
+ * @param {string} config.ipCidrRange - The range of internal addresses that
+ *    are owned by this subnetwork. Provide this property when you create the
+ *    subnetwork. For example, 10.0.0.0/8 or 192.168.0.0/16. Ranges must be
+ *    unique and non-overlapping within a network.
+ * @param {string} config.region - URL of the region where the Subnetwork
+ *    resides.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {module:compute/rule} callback.rule - The created Rule object.
+ * @param {module:compute/operation} callback.operation - An operation object
+ *     that can be used to check the status of the request.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * var name = 'new-subnetwork-name';
+ *
+ * var config = {
+ *   region : 'regions/us-east1',
+ *   ipCidrRange: '10.0.1.0/24'
+ * };
+ *
+ * network.createSubnetwork(name, config,
+ *  function (err, subnetwork, operation, apiResponse) {
+ *   // `subnetwork` is a Subnetwork object.
+ *
+ *   // `operation` is an Operation object that can be used to check the status
+ *   // of the request.
+ * });
+ */
+Network.prototype.createSubnetwork = function(name, config, callback) {
```

Comment:
Yeah, when I was creating this method I thought it was a little strange as well, since none of the other ones work this way. But when writing code to create a subnetwork in my project I wanted to be able to create the subnetwork through my network object. I thought others might want the option as well.